src/main/java/com/synerzip/ExpenseCalculation/ControllerClass.java

```diff
@@ -0,0 +1,28 @@
+package com.synerzip.ExpenseCalculation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+@RestController
+@ControllerAdvice
+@RequestMapping(path = "/app")
+public class ControllerClass {
```

Comment:
path should be as per controller behaviour like if UserController then path "/user"
why ControllerClass? change it as per class Action/behaviour